pages/common/scrapy.md

```diff
@@ -0,0 +1,31 @@
+# scrapy
+
+> Open Source framework for crawling websites written in python.
```

Comment:
Some somewhat unnecessary info here ("open source", "written in python"). I'd suggest removing both.